Tasks/quotes.js

```diff
@@ -1,30 +1,18 @@
 // Change double quotation to open or close quotation
 
-const EMPTY = '';
+'use strict';
 
-quotes = function (s) {
-  res = [];
-  open = false;
-  for (c of s) {
-    if (c === '"') {
-      for (i of c) {
-        if (!open) {
-          res.push('«');
-          open = true;
-        } else {
-          res.push('»');
-          open = false;
-        }
-      }
-    } else {
-      if (c !== '"') {
-        for (i of c) {
-          res.push(i);
-        }
-      }
+const changeQuotes = (string) => {
+  const result = [];
+  let open = false;
+  for (let char of string) {
+    if (char === '"') {
+      !open ? char = '«' : char = '»';
```

Comment:
Incorrect usage of ternary semantics:
```suggestion
      char = !open ? '«' : '»';
```